src/main/kotlin/org/jmailen/gradle/kotlinter/pluginapplier/KotlinSourceSetApplier.kt

```diff
@@ -18,6 +15,6 @@ internal object KotlinJvmSourceSetApplier : SourceSetApplier {
         }
     }
 
-    private fun getSourceSets(project: Project): NamedDomainObjectContainer<KotlinSourceSet> =
-        project.extensions.findByType(KotlinJvmProjectExtension::class.java)!!.sourceSets
+    private fun getSourceSets(project: Project) =
+        project.extensions.getByType(KotlinProjectExtension::class.java).sourceSets
```

Comment:
I started wondering if this might also work for android project 👀 Theoretically it _should_, but I'd have to do a bit more research if AGP doesn't do some additional _magic_